the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
```

Comment:
mysql_db.number7:
Remove commented code.

Reply:
mysql_db.number8:
Use java.sql built in transaction handling.
https://docs.oracle.com/javase/tutorial/jdbc/basics/transactions.html

Reply:
Class dataDriven.Pojo:
Rename this to Student since it models a student.

Reply:
Class dataDriven.Pojo:
I would omit the constructor here. Constructors in POJOs become a maintenance problem later.

Reply:
ParameterDriven,testMethod:
Just pass in a collection of Students rather than each field. Then if more fields are added to Student you won't need to update you test.